app/main.ipynb

```diff
@@ -126,17 +157,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
+   "execution_count": 109,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(0, 1, 4, 7)\n"
+     ]
+    }
+   ],
    "source": [
-    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "from itertools import combinations\n",
+    "\n",
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1], dtype=np.float32)\n",
+    "\n",
+    "for i in range(1, len(array)):\n",
+    "    for comb in combinations(range(len(array)), r=i):\n",
+    "        if array[list(comb)].sum() == 27:\n",
+    "            print(comb)\n",
+    "            break"
```

Comment:
read carefully the task 4 description. You have eather wrong implementation and result
You can skip creating loops here, just pick correct numbers manually, and use **multi-indexing**
`using  multi-indexing you should summarize a subsequence of an array and get the 27 as the result`. So, result also should be 27